src/services/Filter.php

```diff
@@ -94,21 +94,16 @@ public static function title(string $input): string
     }
 
     /**
-     * Remove all non word characters. Used for files saved on the filesystem (pdf, zip, ...)
-     * This code is from https://developer.wordpress.org/reference/functions/sanitize_file_name/
+     * Only allow alphabet characters and numbers
+     * This code is from https://stackoverflow.com/questions/4289905/php-escape-user-input-for-filename/4290182#4290182
      *
      * @param string $input what to sanitize
      * @return string the clean string
      */
     public static function forFilesystem(string $input): string
     {
-        $specialChars = array('?', '[', ']', '/', '\\', '=', '<', '>', ':', ';', ',', "'", '"', '&', '$', '#', '*', '(', ')', '|', '~', '`', '!', '{', '}', '%', '+', chr(0));
-        $input = htmlspecialchars_decode($input, ENT_QUOTES);
-        $input = preg_replace("#\x{00a0}#siu", ' ', $input);
-        $input = str_replace($specialChars, '', $input ?? '');
-        $input = str_replace(array('%20', '+'), '-', $input ?? '');
-        $input = preg_replace('/[\r\n\t -]+/', '-', $input);
-        return trim($input ?? 'file', '.-_');
+        $escaped = preg_replace('/[^A-Za-z0-9_\-]/', '_', $input);
```

Comment:
as I said, don't touch this function!

Reply:
In my environment the download feature does not work without changing this method because of greek letters included in many experiment names. Is it instead of changing the complete function ok for you to add the greek letters to the special character array? Something like that:

```
$specialChars = array('?', '[', ']', '/', '\\', '=', '<', '>', ':', ';', ',', "'", '"', '&', '$', '#', '*', '(', ')', '|', '~', '`', '!', '{', '}', '%', '+', chr(0), 'α', 'β', 'γ', 'δ', [...]. 'ω');
```

Reply:
why greek letters would prevent the download? Except special characters, all UTF-8 characters should be fine. It's totally possible to have filenames with special characters! Can you explain more the "does not work" part?

Reply:
Sure, you can reproduce it with the following title:

```
Some experiment Δ
```

I see the following error when trying to download the file:
```
2019/12/29 12:39:09 [error] 63#63: *2 FastCGI sent in stderr: "PHP message: [2019-12-29T12:39:07.904414+01:00] elabftw.ERROR:  [{"userid":"1"},{"Exception":"[object] (InvalidArgumentException(code: 0): The filename fallback must only contain ASCII characters. at /elabftw/vendor/symfony/http-foundation/HeaderUtils.php:175)"}] []" while reading response header from upstream, client: 172.18.0.1, server: localhost, request: "GET /make.php?what=zip&id=3&type=experiments HTTP/2.0", upstream: "fastcgi://unix:/var/run/php-fpm.sock:", host: "localhost:3148"
```

The same error occurs for other greek letters. This will be the same for other special characters as well and the reason why I initially tried only to allow `a-zA-Z0-9` as file name. In the productive environment most of the experiments have greek letters in the title.

Reply:
what's interesting here is that the exception is thrown by the symfony http foundation component.

See https://github.com/symfony/symfony/issues/9093

Adding another string parameter to [this line](https://github.com/elabftw/elabftw/blob/hypernext/src/controllers/MakeController.php#L157) should fix the issue. See source: https://github.com/symfony/symfony/blob/5.0/src/Symfony/Component/HttpFoundation/HeaderUtils.php#L163

Reply:
Yes, it's really fixing it. I deleted the greek letters from the array and just tried it by using the following:

```
$contentDisposition = $Response->headers->makeDisposition(ResponseHeaderBag::DISPOSITION_ATTACHMENT, $Make->getFileName(), 'fallback-filename');
```
That's fine for me, too. Should I leave it like that? I will push the change in a minute, so you can have a look.